src/components/HeroImage.module.css

```diff
@@ -1,9 +1,5 @@
 .root {
-  padding-bottom: calc(2 / 3 * 100%);
-  position: relative;
-
-  & img {
-    position: absolute;
-    border-radius: 0.25rem;
-  }
+  width: 100%;
+  height: auto;
+  border-radius: 0.25rem;
```

Comment:
With the changes, the hero image is shown in the intrinsic aspect ratio.

We could force a consistent aspect ratio with:

```css
aspect-ratio: 3 / 2;
object-fit: cover;
```